### DISCUSSION TITLE
Add some missing songs from OutRun2SP PS2

### DISCUSSION THREAD

**Discussion by Zzetth** [emoose/OutRun2006Tweaks#213] [Suggestions]
### Describe the feature you'd like to see added

Is possible add the missing songs from PS2 OutRun 2 SP in the in-game menu? The songs are:

- Lift you up!
- Splash Wave SP Remix
- Passing Breeze SP Remix
- Rush a Difficult SP Remix

**Comment by Sonic-TV** [comment]
extra sprites are required if do so.

**Comment by Zzetth** [comment]
> extra sprites are required if do so.

Maybe there a way extract them from PS2 Iso..

**Comment by Tenome** [comment]
Sprites are easy, modifying the in-game menu to add songs is the hard part. Tweaks hasn't really done anything like adding new menus, assets, etc. in-game.
